src/components/PricesChart/index.jsx

```diff
@@ -1,65 +1,79 @@
+/* eslint-disable capitalized-comments */
 /**
  * Module dependencies.
  */
 
-import './index.scss';
-import {
-  CartesianGrid,
-  Line,
-  LineChart,
-  ResponsiveContainer,
-  Tooltip,
-  XAxis,
-  YAxis
-} from 'recharts';
+import * as utils from '@services/utils';
+import { Chart } from 'react-google-charts';
 import React from 'react';
-import useMediaQuery from '@mui/material/useMediaQuery';
+import { useTheme } from '@mui/material';
+import { useTranslation } from 'react-i18next';
 
 /**
  * Function `PriceChart`.
  */
 
 // eslint-disable-next-line react/prop-types
 function PricesChart({ data }) {
-  const isWide = useMediaQuery('(max-width: 1000px)');
-  const isMobile = useMediaQuery('(max-width: 600px)');
+  const { t } = useTranslation();
+  const isDarkMode = useTheme().palette.mode === 'dark';
 
-  let chartWidth = 1000;
-  let chartHeight = 200;
+  const [dimensions] = React.useState({
+    width: Math.min(window.innerWidth, 1000)
+  });
 
-  if (isMobile) {
-    chartWidth = 350;
-    chartHeight = 150;
-  } else if (isWide) {
-    chartWidth = 600;
-  }
+  const options = {
+    backgroundColor: isDarkMode ? 'black' : 'white',
+    hAxis: {
+      textStyle: { color: isDarkMode ? 'white' : 'black' }
+    },
+    legend: {
+      position: 'bottom',
+      textStyle: { color: isDarkMode ? 'white' : 'black' }
+    },
+    vAxis: {
+      textStyle: { color: isDarkMode ? 'white' : 'black' }
+    },
+    width: dimensions.width
+  };
+
+  /**
+   * Create Chart Data.
+   */
+
+  const createChartData = (prices) => {
```

Comment:
Why leave commented out lines on the function?

Reply:
Removed!